### DISCUSSION TITLE
preview failed - showing node.js problems

### DISCUSSION THREAD

**Discussion by MargaretNagawa** [thegetty/quire#255] [Get Help] [answered]
Hello! 

I am not able to see a preview of my project. I have used some of the tips you suggest for such problems including uninstalling node and reinstalling it. Below is the message I get. Thanks for your help. 

Margaret

(node:1991) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open 'config.yml'
    at Object.openSync (fs.js:497:3)
    at Object.readFileSync (fs.js:393:35)
    at B (/usr/local/lib/node_modules/@thegetty/quire-cli/bin/index.js:2:24620)
    at n.value (/usr/local/lib/node_modules/@thegetty/quire-cli/bin/index.js:2:35619)
    at new n (/usr/local/lib/node_modules/@thegetty/quire-cli/bin/index.js:2:34603)
    at /usr/local/lib/node_modules/@thegetty/quire-cli/bin/index.js:2:50209
    at new Promise (<anonymous>)
    at e.value (/usr/local/lib/node_modules/@thegetty/quire-cli/bin/index.js:2:50175)
    at emit (events.js:400:28)
    at Command.<anonymous> (/usr/local/lib/node_modules/@thegetty/quire-cli/bin/index.js:2:122529)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:1991) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:1991) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by geealbers** [comment, marked as answer]
Hi @MargaretNagawa. I'm afraid Quire's error messaging isn't very clear here, but that first line where it says "no such file or directory, open 'config.yml'" is an indicator that you're not in the directory/folder of the new Quire project you started. Use the `cd` command to "change directory" into the project folder and then try `quire preview` again. 

So, for example, if you named your project "my-project" by running the command `quire new my-project`, you would now type `cd my-project` which would move you into that project, and then `quire preview`.

Let us know if that does it!

**Reply by MargaretNagawa** [reply]
Thank you, Greg. 

The problem appears to have been with my project folder. Quire could not find it, so I have deleted that folder and redownloaded the project from GitHub. All seems to be working fine now. 

Thank you,
Margaret
